### PR TITLE
#142 Created Updated fields

### PR DESCRIPTION
**Pull request by vlad-tokarev**

### CODE REVIEW

File: packages/operator/pkg/repository/connection/kubernetes/connection.go [hunk 1]
```diff
@@ -206,6 +207,7 @@ func (kc *k8sConnectionRepository) CreateConnection(connection *connection.Conne
 			Labels:    transformToLabels(connection),
 		},
 		Spec: connection.Spec,
+		Status: connection.Status,
```
Comment (mcdoker18) [inline packages/operator/pkg/repository/connection/kubernetes/connection.go]: We should not allow a user to set the status. This remark is true for all kubernetes repositories.

File: packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go [hunk 2]
```diff
@@ -189,6 +191,8 @@ func (cc *controller) createConnection(c *gin.Context) {
 		return
 	}
 
+	conn.Status.CreatedAt = &metav1.Time{Time: time.Now()}
```
Comment (mcdoker18) [inline packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go]: I thought that we will fill both CreatedAt and UpdatedAt fields during creation of entity. What do you think about it?

File: packages/operator/pkg/webserver/routes/v1/deployment/model_deployment.go [hunk 3]
```diff
@@ -135,6 +137,8 @@ func (mdc *ModelDeploymentController) createMD(c *gin.Context) {
 		return
 	}
 
+	md.Status.CreatedAt = &metav1.Time{Time: time.Now()}
```
Comment (mcdoker18) [inline packages/operator/pkg/webserver/routes/v1/deployment/model_deployment.go]: Unit tests for new functionality?

File: packages/sdk/odahuflow/sdk/models/toolchain_integration_status.py [hunk 4]
```diff
@@ -15,16 +15,27 @@ class ToolchainIntegrationStatus(Model):
     Do not edit the class manually.
```
Comment (mcdoker18) [inline packages/sdk/odahuflow/sdk/models/toolchain_integration_status.py]: I'm a little curious. How do the new fields look like in JSON format?

Comment (vlad-tokarev) [reply packages/sdk/odahuflow/sdk/models/toolchain_integration_status.py]: "status": {
      "createdAt": "2020-03-04T22:04:44Z"
    }

Comment (vlad-tokarev) [reply packages/sdk/odahuflow/sdk/models/toolchain_integration_status.py]: It seems that ISO dateformat is definitely OK for rest api

File: packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go (hunk 2 above)
Comment (vlad-tokarev) [reply packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go]: what advantages a user will get in that case? Or you think that this logic of **updated field** is just more straightforward?

Comment (vlad-tokarev) [reply packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go]: added

**Pull request merged**